core-chart/templates/postcreatehooks/install-status-addon.yaml

```diff
@@ -0,0 +1,123 @@
+{{- if .Values.InstallPCHs }}
+apiVersion: tenancy.kflex.kubestellar.org/v1alpha1
+kind: PostCreateHook
+metadata:
+  name: install-status-addon
+  labels:
+    kflex.kubestellar.io/cptype: its
+spec:
+  templates:
+  - apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: "{{"{{.HookName}}"}}"
+    spec:
+      template:
+        spec:
+          initContainers:
+          - name: wait-for-kubeconfig
+            image: quay.io/kubestellar/kubectl:{{.Values.KUBECTL_VERSION}}
```

Comment:
Could we use `busybox` instead of our kubectl image? We want to work towards removing `bash` from our kubectl image.